### PR TITLE
[web] project delete confirm content

### PR DESCRIPTION
**Pull request by elliotmessi**

### CODE REVIEW

File: ymir/web/src/locales/modules/project.ts [hunk 1]
```diff
@@ -89,6 +89,8 @@ const project = {
   'project.tag.mining': { en: 'Mining Dataset {version}', cn: '挖掘集 {version}', },
   'project.tag.model': { en: 'Initial Model {version}', cn: '初始模型 {version}', },
   'iteration.tag.round': { en: 'Round {round}', cn: '迭代{round}', },
+  'project.del.confirm.content': { en: '删除项目会将项目中的所有资源（数据集、模型）删除，请谨慎操作！', 
+    cn: 'Delete project will remove all datasets and models in project, please delete after careful consideration!', },
```
Comment (phoenix-xhuang) [inline ymir/web/src/locales/modules/project.ts]: Remove all datasets and models in this project, confirm?

Comment (elliotmessi) [reply ymir/web/src/locales/modules/project.ts]: 这里目标是删除项目，不是删除数据集和模型，资源被清除只是附带的，不能这么提示吧

Comment (phoenix-xhuang) [reply ymir/web/src/locales/modules/project.ts]: 那可以改为
This will remove all datasets and models in current project, confirm?

原文太冗长了

**Review by phoenix-xhuang — APPROVED** [review]

File: ymir/web/src/locales/modules/project.ts (hunk 1 above)
Comment (elliotmessi) [reply ymir/web/src/locales/modules/project.ts]: fixed

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**